pkg/cmd/server/api/v1/types.go

```diff
@@ -881,3 +888,12 @@ type LDAPQuery struct {
 	// Filter is a valid LDAP search filter that retrieves all relevant entries from the LDAP server with the base DN
 	Filter string `json:"filter"`
 }
+
+type AdmissionConfig struct {
+	// PluginConfig allows specifying a configuration file per admission control plugin
+	PluginConfig map[string]string `json:"pluginConfig"`
```

Comment:
> Correct. My assumption is that we need another admission controller to set forcePull for all builds. @deads2k is suggesting we enable them by default, but make the default configuration have them perform no-ops.

Having both master-config.yaml flags to turn on some admission plugins and then having a separate set of overrides seems pretty ugly, but I'm assuming that's the alternative you're considering?